spec/support/aruba_support.rb

```diff
@@ -10,7 +10,7 @@ module ArubaLoader
   let(:stderr) { StringIO.new }
   let(:stdout) { StringIO.new }
 
-  attr_reader :last_cmd_stdout, :last_cmd_stderr
+  attr_reader :last_cmd_stdout, :last_cmd_stderr, :last_cmd_exitstatus
```

Comment:
I was influenced by [`$?.exitstatus`](http://ruby-doc.org/core-2.2.0/Process/Status.html), but `last_cmd_exit_status` sounds better to me too.